src/translib/transformer/xfmr_showtech.go

```diff
@@ -0,0 +1,69 @@
+////////////////////////////////////////////////////////////////////////////////
+//                                                                            //
+//  Copyright 2019 Dell, Inc.                                                 //
+//                                                                            //
+//  Licensed under the Apache License, Version 2.0 (the "License");           //
+//  you may not use this file except in compliance with the License.          //
+//  You may obtain a copy of the License at                                   //
+//                                                                            //
+//  http://www.apache.org/licenses/LICENSE-2.0                                //
+//                                                                            //
+//  Unless required by applicable law or agreed to in writing, software       //
+//  distributed under the License is distributed on an "AS IS" BASIS,         //
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  //
+//  See the License for the specific language governing permissions and       //
+//  limitations under the License.                                            //
+//                                                                            //
+////////////////////////////////////////////////////////////////////////////////
+
+package transformer
+
+import (
+//	"bytes"
+//	"errors"
+//	"fmt"
+	"encoding/json"
+	"translib/tlerr"
+	"translib/db"
+	"github.com/golang/glog"
+)
+
+func init() {
+	XlateFuncBind("rpc_showtech_cb", rpc_showtech_cb)
+}
+
+var rpc_showtech_cb RpcCallpoint = func(body []byte, dbs [db.MaxDB]*db.DB) ([]byte, error) {
+	var err error
+    var output string
```

Comment:
Please do fix the alignment for the whole file.

Reply:
There were many lines in this file with mixed tabs and spaces. I have replaced all of the tabs with spaces and re-verified the resulting indentation and alignment.